src/settings/VoucherNumber/SettingsVoucherNumber.js

```diff
@@ -49,14 +71,31 @@ class SettingsVoucherNumber extends Component {
     return get(resources, ['voucherNumber', 'records', 0, 'sequenceNumber'], '');
   };
 
+  getInitialValues = (config) => {
+    const sequenceNumber = this.getStartSequenceNumber();
+    const configSetting = getConfigSetting(config);
+
+    return {
+      ...configSetting,
+      sequenceNumber,
```

Comment:
what is the reason to mix it here, if you pass it separately? `firstSequenceNumber={sequenceNumber}`
also, `firstSequenceNumber` is kind of confusing.

Reply:
Fixed